### PR TITLE
add fedora_schema

### PR DESCRIPTION
**Pull request by geekscruff**
Added fedora_schema for core_metadata, basic_metadata and a handful of other predicates scattered throughout Hyrax.

Added it to a separate class for readability.

Question: should I test these go into Fedora, or not?

### CODE REVIEW

File: lib/hyrax/configuration.rb [hunk 1]
```diff
@@ -417,9 +417,7 @@ def uploader
 
     attr_writer :fedora_schema
     def fedora_schema
-      @fedora_schema ||= {
-        depositor: RDF::URI("http://id.loc.gov/vocabulary/relators/dpt")
-      }
+      @fedora_schema ||= Hyrax::FedoraSchema.fedora_schema
```
Comment (jcoyne) [inline lib/hyrax/configuration.rb]: as this is already a class attribute. I see no need to memoize this any longer.

Comment (jcoyne) [reply lib/hyrax/configuration.rb]: Could just be `delegate :fedora_schema, :fedora_schema=, to: Hyrax::FedoraSchema`

Comment (geekscruff) [reply lib/hyrax/configuration.rb]: that won't mess up overriding it locally will it?

Comment (geekscruff) [reply lib/hyrax/configuration.rb]: Done. Learning new things every day ... today 'memoize'!

**Pull request merged**